backend/app/core/models.py

```diff
@@ -0,0 +1,13 @@
+from typing import Optional
+
+from pydantic import BaseModel
+
+
+class ServerSessionData(BaseModel):
+    """Pydantic model used to validate server session data.
+    Created first during user login.
+    """
```

Comment:
This is a critical error on the built in Cadair linter. Please reformat as:

```suggestion
    """
    Pydantic model used to validate server session data.
    Created first during user login.
    """
```